scheduler/src/cook/mesos/api.clj

```diff
@@ -243,7 +241,7 @@
                   :priority (or priority util/default-job-priority)
                   :max-retries max_retries
                   :max-runtime (or max_runtime Long/MAX_VALUE)
-                  :ports (or ports [])
+                  :ports (or ports 0)
```

Comment:
The original fenzo branch attempted to change the the type of the Datomic attribute "port", which did cause issues.  This branch leaves the original attribute there (though it's no longer used), and introduces a new attribute "ports", which is what would populate this part of the API response.